.github/workflows/release.yml

```diff
@@ -7,40 +7,43 @@ env:
 
 name: Release
 on:
-  workflow_dispatch: null
+  workflow_dispatch:
 jobs:
   Version:
     outputs:
-      created: ${{ steps.daily-version.outputs.created }}
-      version: ${{ steps.daily-version.outputs.version }}
+      created: ${{ env.DAILY_VERSION_CREATED }}
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-        with:
-          fetch-depth: 20
```

Comment:
Only needed by the previous releaser https://github.com/fregante/release-with-changelog